src/react/components/VectorStyle/index.js

```diff
@@ -0,0 +1,2 @@
+export { VectorNameInput } from './VectorNameInput';
```

Comment:
Only VectorStyleModal uses VectorNameInput so we could remove export for the input from here?

Reply:
Fixed this.